doc/attactions.md

```diff
@@ -0,0 +1,50 @@
+# ITS Attractions
+
+**Maclisp** &mdash; historical Lisp for ITS and Multics, and the base
+for a family of dialects that consolidated as the Common Lisp standard.
+It was used to bootstrap **Scheme**.
+
+**SHRDLU** &mdash; infamous natural language input and block stacking
+robot simulation, written in Maclisp.
+
+**Lisp machines** &mdash; were bootstrapped from ITS and Maclisp.
+
+**C compiler** &mdash; written by Alan Snyder; possibly the first
+created outside Bell Labs.
+
+**Macsyma** &mdash; symbolic manipulation program that was written in
+Maclisp and made available to researchers on the Arpanet.  Macsyma was
+so important a dedicated PDP-10 was purchased for it, not just once,
+but twice.
+
+**Emacs** &mdash; was developed on ITS as a set of **TECO** macros.
+
+**The Magic Switch** &mdash; a piece of hacker lore chronicled in Guy
+Steele's book *Hacker's Dictionary*.  The book comes from the
+**JARGON** file.
+
+**Logo** &mdash; educational programming language began at BBN but
+greatly enhanced at MIT.  ITS hosts many versions for the PDP-10,
+PDP-11, Apple II, etc.  A **Small ITS** timesharing system was written
+to run Logo on a PDP-11/45.  Marvin Minsky designed the **2500**, a
+dual text and vector display minicomputer for running Logo.
+
+**CLU** &mdash; a programming language by Barbara Liskov that
+introduced abstract data types way ahead of its times.  The CLU group
+developed the first version of the X Window System.
+
+**Zork** &mdash; was written in **MDL**/**Muddle**, the major
+programming language on the Dynamic Modeling PDP-10.  Zork was
+inspired by (**Colossal Cave**) Advent**ure, which is also available
```

Comment:
What is the double ** in the middle of Adventure?

Reply:
Formatting gone wrong.  Fixed.